JDStatusBarNotification.xcodeproj/xcshareddata/xcschemes/JDStatusBarNotification.xcscheme

```diff
@@ -0,0 +1,82 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
do we need this file to be checked in?

Reply:
Yes, Carthage needs a (shared) framework scheme to be able to build.